packages/config/config/devices/0x015f/mh5-2a.json

```diff
@@ -0,0 +1,389 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH5-2A",
+	"description": "Fan Coil Thermostat",
+	"devices": [
+		{
+			"productType": "0xf501",
+			"productId": "0x7102",
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Temperature Unit",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celsius",
+					"value": 0
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 0
```

Comment:
@blhoward2 this is supposed to be 1, right?

Reply:
I confirm this should be 1 yeah, based on manual

Reply:
I'm fine if you want to make Celsius the default.

Reply:
I meant the option value, we have zero twice.